### DISCUSSION TITLE
Tablet does not work when OpenTabletDriver is active

### DISCUSSION THREAD

**Discussion by ALANVF** [ppy/osu#16967] [Q&A]
(Note: I'll also be reporting this on OpenTabletDriver's github, since this didn't used to happen and is probably part of a recent OTD update)

I'm using a Wacom Bamboo Fun CTE-650, and I'm having an unusual issue where my tablet is completely disabled when OpenTabletDriver is active. Once whatever app that's using OTD is closed (in this case, osu!lazer) it returns back to normal. I was messing around with the tablet settings a bit, and ended up getting a weird error so I thought I'd drop the logs here.

I cannot reproduce it for some reason, but basically I was unplugging/re-plugging the tablet while enabling/disabling the tablet detection setting.

(fun stuff starts on line 132)

[runtime.log](https://github.com/ppy/osu/files/8121352/runtime.log)

**Comment by TheFantasticLoki** [comment]
You do use OTD with stable correct? As in the standalone driver and not the one included in osu!lazer. If you have both running they conflict with each other so if you already use the normal driver when playing stable just untick the enable tablet option inside osu!lazer then restart osu!lazer and hopefully your problem is resolved.

**Reply by peppy** [reply]
This is not relevant. Check their log file. It should not "conflict" in the way that is seen there.

**Reply by ALANVF** [reply]
No I don't use OTD in stable
